SU-HSDO/suhumsci
# SHS-5054: CKEditor blockquote preview

# READY FOR REVIEW

## Summary
Preview of the blockquote looks closer to how it will appear when the page is saved and the content is rendered.

## Urgency
low

## Steps to Test
1. Go to any page and edit it
2. In the wysiwyg add a blockquote
3. Check the preview looks closer to how it looks once the page is saved and the content is rendered

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
